js-apps/meep-frontend/src/js/state/ui/index.js

```diff
@@ -60,7 +60,11 @@ const initialState = {
   replayFileDesc: '',
   sandbox: '',
   sandboxes: [],
-  sandboxCfg: {}
+  sandboxCfg: {},
+  helpMenuDisplay: false,
+  signInStatus: STATUS_SIGNIN_UNKNOWN,
+  signInUsername: '',
+  activeTabIndex: 0
```

Comment:
When upgrading from an older version, this initial state is not used and therefore the default values are not set. Does the code take this into account?

For example, to verify this install AvantEDGE release 1.6.1, clear the frontend local storage and run some basic operations using the frontend. Then install this latest code base and see if it works as expected (e.g. defaults to Home page, shows the sign-in button, etc.)

Reply:
Added versioning in localstorage. So whenever we have some update like this, we just have to update the version package.json